tests/data/MerchFactory.ts

```diff
@@ -33,6 +34,10 @@ export class MerchFactory {
     return MerchFactory.optionMetadataWith(...substitutes);
   }
 
+  public static createOrderPickupEvents(n: number): OrderPickupEventModel[] {
```

Comment:
Since we just deleted a bunch of unnecessary methods, let's only create `fakeOrderPickupEvent` for now and delete `createXXX` and `XXXwith`. We can add them back if there's an actual need for them.